src/TipBox.vue

```diff
@@ -167,15 +181,23 @@
             return '<i class="fas fa-lightbulb"></i>';
           case 'definition':
             return '<i class="fas fa-atlas"></i>';
-          default:
-            return '<i class="fas fa-exclamation"></i>';
         }
       },
       iconStyle() {
         if (this.iconSize) {
           return `fa-${this.iconSize}`;
         }
         return '';
+      },
+      hasBackground() {
+        if (this.noBackground) {
+          return 'no-background';
+        }
+      },
+      hasBorder() {
+        if (this.noBorder) {
+          return 'no-border';
+        }
```

Comment:
Probably `backgroundStyle` and `borderStyle` are more consistent names?

Reply:
It would be deceiving to say these style the backgrounds and border, since that's what `customStyle` does. I will rename it `noBackgroundStyle` and `noBorderStyle` because they enforce the styles of having no background or no border.